### PR TITLE
Migration: mention the removal of `.rounded-sm/lg` utilities

### LINKED ISSUE #32734: [Docs] v5 migration guide doesn't mention removal of .rounded-sm/lg
It looks like #31687 removed `.rounded-sm` and `.rounded-lg` and introduced `.rounded-0`-`.rounded-3`

This change totally makes sense, but I think maybe it's missing from the [Migrating to v5](https://getbootstrap.com/docs/5.0/migration/) docs?

### PR DESCRIPTION
**Pull request by imkalashsharma**
Issue - [Docs] v5 migration guide doesn't mention removal of .rounded-sm/lg

Added information about the removal of `.rounded-sm` and `.rounded-lg`.
And addition of `.rounded-0` to `.rounded-3`.

Screenshot (227)

Fixes #32734

### CODE REVIEW

File: site/content/docs/5.0/migration.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ toc: true
 ### Sass
 
 #### Utilities
-
+- Removed `.rounded-sm` and `rounded-lg` and introduced `.rounded-0` to `.rounded-3`.
```
Comment (mdo) [inline site/content/docs/5.0/migration.md:15]: This should be under the v5.0.0-alpha3 migration guide, not the beta2 one. https://getbootstrap.com/docs/5.0/migration/#v500-alpha3

Can you please move it, and add a link to the PR? https://github.com/twbs/bootstrap/pull/31687

Comment (k99sharma) [reply site/content/docs/5.0/migration.md:15]: Sure thing !

File: site/content/docs/5.0/migration.md [hunk 2]
```diff
@@ -127,6 +126,7 @@ Breakpoints specific variants are consequently renamed too (eg. `.text-md-start`
   - Renamed `.font-style-*` utilities as `.fst-*` for brevity and consistency.
 - Added `.d-grid` to display utilities
 - Added new `gap` utilities (`.gap`) for CSS Grid layouts
+ - Removed `.rounded-sm` and `rounded-lg` and introduced `.rounded-0` to `.rounded-3`.
```
Comment (ffoodd) [inline site/content/docs/5.0/migration.md:129]: Unwanted whitespace:

```suggestion
- Removed `.rounded-sm` and `rounded-lg` and introduced `.rounded-0` to `.rounded-3`.
```

**Review by ffoodd — CHANGES_REQUESTED** [review]
Thanks for your PR :heart:

File: site/content/docs/5.0/migration.md (hunk 2 above)
Comment (ffoodd) [reply site/content/docs/5.0/migration.md:129]: Oh and you still miss the link to #31687 :)

Comment (k99sharma) [reply site/content/docs/5.0/migration.md]: This is my first time contributing so I didn't quite get it the first time where to put the link. I just made the changes please check if it's still not right.  😅

Comment (ffoodd) [reply site/content/docs/5.0/migration.md]: Don't worry, you did well :+1:

File: site/content/docs/5.0/migration.md [hunk 3]
```diff
@@ -127,6 +126,7 @@ Breakpoints specific variants are consequently renamed too (eg. `.text-md-start`
   - Renamed `.font-style-*` utilities as `.fst-*` for brevity and consistency.
 - Added `.d-grid` to display utilities
 - Added new `gap` utilities (`.gap`) for CSS Grid layouts
+ - Removed `.rounded-sm` and `rounded-lg` and introduced `.rounded-0` to `.rounded-3`. [See #31687](https://github.com/twbs/bootstrap/pull/31687)
```
Comment (ffoodd) [inline site/content/docs/5.0/migration.md:129]:
```suggestion
- Removed `.rounded-sm` and `rounded-lg`, and introduced `.rounded-0` to `.rounded-3`. [See #31687](https://github.com/twbs/bootstrap/pull/31687).
```

**Review by ffoodd — CHANGES_REQUESTED** [review]

**Review by ffoodd — APPROVED** [review]

File: site/content/docs/5.0/migration.md (hunk 2 above)
Comment (k99sharma) [reply site/content/docs/5.0/migration.md]: Thanks !

**Pull request merged**